tests/ImageSharp.Tests/Common/NumericsTests.cs

```diff
@@ -97,5 +97,25 @@ public void DivideCeil_RandomValues(int seed, int count)
                 Assert.True(expected == actual, $"Expected: {expected}\nActual: {actual}\n{value} / {divisor} = {expected}");
             }
         }
+
+        private static bool IsOutOfRange_ReferenceImplementation(int value, int min, int max) => value < min || value > max;
+
+        [Theory]
+        [InlineData(1, 100)]
+        public void IsOutOfRange(int seed, int count)
+        {
+            var rng = new Random(seed);
```

Comment:
Instead of random numbers to test, I'd rather test edge-cases.

E.g.
```
min := 7
max := 92

value = { int.MinValue, -1, 0, 1, 6, 7, 8, 91, 92, 93, int.MaxValue }
```
then the whole range is covered.

Reply:
Agreed, fixed.